testnets/nomictestnet/assetlist.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "../assetlist.schema.json",
```

Comment:
need two (up) directories to reference the schema's location.
"../../assetlist.schema.json"